apps/explorer/src/pages/validator/ValidatorDetails.tsx

```diff
@@ -67,12 +189,14 @@ function ValidatorDetails(): JSX.Element {
                 content={
                     <div className="mb-10">
                         <InfoBox
-                            title="Failed to load validator data"
-                            supportingText={`No validator data found for ${id}`}
+                            title="Inactive validator"
```

Comment:
What if a validator does not exist + is not inactive ? I think it would make sense to still render "Failed to load validator data" and render "Inactive validator" if its really inactive